Description: Set AuxiliaryFiles and UseCachedExampleOutput options to
 true for several packages whose examples are cached because
 generating them has been known to cause build failures on various
 architectures.  See debian/scripts/generate-examples.m2.
Author: Doug Torrance <dtorrance@debian.org>
Forwarded: not-needed
Last-Update: 2021-02-20

--- a/M2/Macaulay2/packages/Macaulay2Doc.m2
+++ b/M2/Macaulay2/packages/Macaulay2Doc.m2
@@ -6,6 +6,7 @@
 newPackage ("Macaulay2Doc",
      InfoDirSection => "Macaulay2 and its packages", 
      AuxiliaryFiles => true,
+     UseCachedExampleOutput => true,
      Headline => "Macaulay2 documentation", 
      Authors => {
 	  {Name => "Daniel R. Grayson", Email => "dan@math.uiuc.edu", HomePage => "http://www.math.uiuc.edu/~dan/"},
--- a/M2/Macaulay2/packages/SemidefiniteProgramming.m2
+++ b/M2/Macaulay2/packages/SemidefiniteProgramming.m2
@@ -19,6 +19,7 @@
     Headline => "semidefinite programming",
     Keywords => {"Real Algebraic Geometry", "Interfaces"},
     Configuration => {"CSDPexec"=>"","MOSEKexec"=>"mosek","SDPAexec"=>"sdpa","DefaultSolver"=>null},
+    UseCachedExampleOutput => true,
     AuxiliaryFiles => true,
     PackageExports => {"NumericalAlgebraicGeometry"}
 )
--- a/M2/Macaulay2/packages/AssociativeAlgebras.m2
+++ b/M2/Macaulay2/packages/AssociativeAlgebras.m2
@@ -13,6 +13,7 @@
     Keywords => {"Noncommutative Algebra"},
     PackageImports =>{"Complexes"},
     PackageExports =>{"IntegralClosure"},
+    UseCachedExampleOutput => true,
     AuxiliaryFiles => true
     )
 
--- a/M2/Macaulay2/packages/SuperLinearAlgebra.m2
+++ b/M2/Macaulay2/packages/SuperLinearAlgebra.m2
@@ -14,6 +14,8 @@
     }, 
     Headline => "computations related to supermatrices", 
     DebuggingMode => false,
+    AuxiliaryFiles => true,
+    UseCachedExampleOutput => true
     Keywords => {"Linear Algebra"}
 )
 
--- a/M2/Macaulay2/packages/ThreadedGB.m2
+++ b/M2/Macaulay2/packages/ThreadedGB.m2
@@ -15,6 +15,8 @@
     },
     Keywords => {"Groebner Basis Algorithms"},
     Headline => "Compute a Groebner basis using the classical Buchberger with multiple threads",
+    AuxiliaryFiles => true,
+    UseCachedExampleOutput => true,
     Certification => {
 	 "journal name" => "The Journal of Software for Algebra and Geometry",
 	 "journal URI" => "https://msp.org/jsag/",
